### PR TITLE
[bitnami/grafana-loki] Update README to include persistence limitation

### LINKED ISSUE #12916: [bitnami/grafana-loki] Is filesystem storage supported?
### Name and Version

bitnami/grafana-loki: 2.4.2

### What steps will reproduce the bug?

I've been trying to get Loki working without an object storage backend. I'm not sure if the chart supports this. The current behaviour I'm experiencing is that I'm unable to query any logs that's older than 1hr.

I can't locate anything in the README on this as well.

For what it's worth, here's the Loki configuration.

```yaml
auth_enabled: true
chunk_store_config:
  chunk_cache_config:
    memcached:
      batch_size: 100
      parallelism: 100
    memcached_client:
      addresses: dns+observability-stack-memcachedchunks:11211
      consistent_hash: true
  max_look_back_period: 0s
compactor:
  shared_store: filesystem
distributor:
  ring:
    kvstore:
      store: memberlist
frontend:
  compress_responses: true
  log_queries_longer_than: 5s
  tail_proxy_url: http://observability-stack-loki-querier:3100
frontend_worker:
  frontend_address: observability-stack-loki-query-frontend:9095
ingester:
  chunk_block_size: 262144
  chunk_encoding: snappy
  chunk_idle_period: 30m
  chunk_retain_period: 1m
  lifecycler:
    ring:
      kvstore:
        store: memberlist
      replication_factor: 1
  max_transfer_retries: 0
  wal:
    dir: /bitnami/grafana-loki/wal
limits_config:
  enforce_metric_name: false
  max_cache_freshness_per_query: 10m
  per_tenant_override_config: /etc/tenants/overrides.yaml
  reject_old_samples: true
  reject_old_samples_max_age: 168h
  retention_period: 744h
  split_queries_by_interval: 15m
memberlist:
  join_members:
  - observability-stack-loki-gossip-ring
query_range:
  align_queries_with_step: true
  cache_results: true
  max_retries: 5
  results_cache:
    cache:
      memcached_client:
        addresses: dns+observability-stack-memcachedfrontend:11211
        consistent_hash: true
        max_idle_conns: 16
        timeout: 500ms
        update_interval: 1m
ruler:
  alertmanager_url: https://alertmanager.xx
  external_url: https://alertmanager.xx
  ring:
    kvstore:
      store: memberlist
  rule_path: /tmp/loki/scratch
  storage:
    local:
      directory: /bitnami/grafana-loki/conf/rules
    type: local
schema_config:
  configs:
  - from: "2020-10-24"
    index:
      period: 24h
      prefix: index_
    object_store: filesystem
    schema: v11
    store: boltdb-shipper
server:
  http_listen_port: 3100
storage_config:
  boltdb_shipper:
    active_index_directory: /bitnami/grafana-loki/loki/index
    cache_location: /bitnami/grafana-loki/loki/cache
    cache_ttl: 168h
    shared_store: filesystem
  filesystem:
    directory: /bitnami/grafana-loki/chunks
  index_queries_cache_config:
    memcached:
      batch_size: 100
      parallelism: 100
    memcached_client:
      addresses: dns+observability-stack-memcachedindexqueries:11211
      consistent_hash: true
      service: http
table_manager:
  retention_deletes_enabled: false
  retention_period: 0s
```

### Are you using any custom parameters or values?

_No response_

### What is the expected behavior?

Show logs that are older than 1hr

### What do you see instead?

Only show logs within the hour

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by k-jingyang**
### Description of the change

This changes adds a description on the limitation of the chart, specifically on the topic of local filesystem persistence.

### Benefits

Other users will be made aware of the limitation.

### Possible drawbacks

NIL

### Applicable issues

Fixes #12916

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/grafana-loki/README.md [hunk 1]
```diff
@@ -1229,10 +1229,16 @@ It is strongly recommended to use immutable tags in a production environment. Th
 Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
 ### Loki configuration
 
+
+
```
Comment (carrodher) [inline bitnami/grafana-loki/README.md:1233]: Those lines shouldn't be needed
```suggestion
```

File: bitnami/grafana-loki/README.md [hunk 2]
```diff
@@ -1229,10 +1229,16 @@ It is strongly recommended to use immutable tags in a production environment. Th
...
 The loki configuration file `loki.yaml` is shared across the different components: `distributor`, `compactor`, `ingester`, `querier` and `queryFrontend`. This is set in the `loki.configuration` value. Check the official [Loki Grafana documentation](https://grafana.com/docs/loki/latest/configuration/) for the list of possible configurations.
 
 ## Persistence
 
+### Limitation
+This chart does not function fully when using local filesystem as a persistence store. When using a local filesystem as a persistence store, querying will not be possible (or limited to the ingesters' in-memory caches). For a fully functional deployment of this helm chart, an object storage backend is required.
+
+### Data
 The [Bitnami grafana-loki](https://github.com/bitnami/containers/tree/main/bitnami/grafana-loki) image stores the grafana-loki `ingester` data at the `/bitnami` path of the container. Persistent Volume Claims are used to keep the data across deployments.
```
Comment (carrodher) [inline bitnami/grafana-loki/README.md:1242]: Usually, there is a blank line after the titles
```suggestion
### Limitation

This chart does not function fully when using local filesystem as a persistence store. When using a local filesystem as a persistence store, querying will not be possible (or limited to the ingesters' in-memory caches). For a fully functional deployment of this helm chart, an object storage backend is required.

### Data

The [Bitnami grafana-loki](https://github.com/bitnami/containers/tree/main/bitnami/grafana-loki) image stores the grafana-loki `ingester` data at the `/bitnami` path of the container. Persistent Volume Claims are used to keep the data across deployments.
```

**Review by carrodher — COMMENTED** [review]
Thanks for your contribution, please take a look at the inline comments about some format issues.
Apart from that, I will move the PR to "In progress" so someone else in the team can take a look at the content of the changes

**Review by fmulero — APPROVED** [review]
Thanks a lot @k-jingyang. LGTM

**Pull request merged**